IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -78,51 +76,47 @@
     <hr />
     <div>
         <table border="1" width="90%">
-            <tr>
-                <th>ID</th>

```

Comment:
I removed this Id element from both the tables as because they are internal to the dataset being displayed. From security point of view its not safe to display Ids on the HTML page. Also, displaying Id will not benefit the user of the website.